pinot-spi/src/main/java/org/apache/pinot/spi/utils/JsonUtils.java

```diff
@@ -719,4 +721,19 @@ private static void addFieldToPinotSchema(Schema pinotSchema, DataType dataType,
       }
     }
   }
+
+  public static List<Map<String, String>> flatten(String jsonString, JsonIndexConfig jsonIndexConfig)
```

Comment:
@wirybeaver : what about the other `JsonUtils#flatten` methods? Are they still being used? I see them only being used in tests (on a different branch). If so let's make them protected.

Reply:
You are right. Make the method protected